feathr_project/feathr/api/app/routes/project.py

```diff
@@ -0,0 +1,36 @@
+
+from __future__ import annotations
+from typing import Optional
+
+from fastapi import APIRouter, HTTPException, Response,status
+from app.backends.purview_backend.purview_backend import PurviewBackend
+
+from app.core.configs import *
+from app.core.error_handling import verifyCode
+from app.models.features import Features
+
```

Comment:
Seems I cannot comment on empty files. Are offline_faeture.py and online_faeture.py required? Looks like they are emtpy files